ros2_batch_job/packaging.py

```diff
@@ -111,6 +111,24 @@ def build_and_test_and_package(args, job):
             print('no overlay packages specified')
         print('# END SUBSECTION')
 
+    # Only on Linux update the prefix level setup files
+    # to exclude the chained ROS 1 workspace used for the ros1_bridge
+    if args.os == 'linux':
+        print('# BEGIN SUBSECTION: remove ROS 1 underlay')
+        for setup_file in glob.glob(os.path.join(args.installspace, 'setup.*')):
+            with open(setup_file, 'rb') as h:
+                content = h.read()
+            lines = content.splitlines()
+            start_index = lines.index(b'# source chained prefixes')
```

Comment:
Is there a case where the packaging job is triggered in a way that the underlay doesn't exist? Then this line would fail.

Reply:
These searched for lines are always present in the setup file - even if no underlay is used. See https://github.com/colcon/colcon-core/blob/f742e9261bc1f75e78469cbc48e2963ceb82844f/colcon_core/shell/template/prefix_chain.sh.em#L33-L42

Reply:
My workspace locally doesn't have the `# source chained prefixes` line.

https://github.com/colcon/colcon-core/blob/f742e9261bc1f75e78469cbc48e2963ceb82844f/colcon_core/shell/template/prefix_chain.sh.em#L31

Reply:
Good point. Updated in 96328ef1713b517f5b04a0a29f387e7b471b6092.